lib/dotcom_web/components/trip_planner/input_form.ex

```diff
@@ -75,6 +75,7 @@ defmodule DotcomWeb.Components.TripPlanner.InputForm do
             :if={show_datepicker?(f)}
             module={DatePicker}
             config={datepicker_config()}
+            locale={Gettext.get_locale(Dotcom.Gettext)}
```

Comment:
nit: Might be better to pass this in as a prop from the parent LiveView (which has the locale already in its assigns via a server-side hook). Just not sure how this version would behave after we get around to implementing language switching.